### PR TITLE
Benchmarks: Code Revision - Extract base class for gemm flops microbenchmark

### PR DESCRIPTION
**Pull request by yukirora**
**Description**
Extract base class for gemm flops microbenchmark.

**Major Revision**
- extract base class for gemm flops microbenchmark and add related test.
- revise gemm_flops_performance for cuda.

### CODE REVIEW

File: superbench/benchmarks/micro_benchmarks/__init__.py [hunk 1]
```diff
@@ -21,5 +21,5 @@
 __all__ = [
     'MicroBenchmark', 'MicroBenchmarkWithInvoke', 'ShardingMatmul', 'ComputationCommunicationOverlap', 'KernelLaunch',
     'CublasBenchmark', 'CudnnBenchmark', 'GemmFlopsBenchmark', 'CudaGemmFlopsBenchmark', 'MemBwBenchmark',
-    'CudaMemBwBenchmark', 'DiskBenchmark', 'IBLoopbackBenchmark', 'CudaNcclBwBenchmark'
+    'CudaMemBwBenchmark', 'DiskBenchmark', 'IBLoopbackBenchmark', 'CudaNcclBwBenchmark', 'RocmMemBwBenchmark'
```
Comment (cp5555) [inline superbench/benchmarks/micro_benchmarks/__init__.py:24]: `RocmMemBwBenchmark` in this PR?

File: superbench/benchmarks/micro_benchmarks/gemm_flops_performance.py [hunk 2]
```diff
@@ -8,10 +8,10 @@
 from superbench.common.utils import logger
```
[the remark this answers is not in the document]
Comment (yukirora) [reply superbench/benchmarks/micro_benchmarks/gemm_flops_performance.py:8]: renamed

File: superbench/benchmarks/micro_benchmarks/__init__.py (hunk 1 above)
Comment (yukirora) [reply superbench/benchmarks/micro_benchmarks/__init__.py:24]: it's from main, not in this pr

Comment (cp5555) [reply superbench/benchmarks/micro_benchmarks/__init__.py:24]: Got it. Solved.

**Pull request merged**